docs/books/user-guide/managing-using-qdmanage.adoc

```diff
@@ -53,30 +53,62 @@ For example, the following command executes a query operation on a router, and t
 $ qdmanage query --type listener
 [
   {
-    "stripAnnotations": "both", 
-    "addr": "127.0.0.1", 
-    "multiTenant": false, 
-    "requireSsl": false, 
-    "idleTimeoutSeconds": 16, 
-    "saslMechanisms": "ANONYMOUS", 
-    "maxFrameSize": 16384, 
-    "requireEncryption": false, 
-    "host": "0.0.0.0", 
-    "cost": 1, 
-    "role": "normal", 
-    "http": false, 
-    "maxSessions": 32768, 
-    "authenticatePeer": false, 
-    "type": "org.apache.qpid.dispatch.listener", 
-    "port": "amqp", 
-    "identity": "listener/0.0.0.0:amqp", 
+    "stripAnnotations": "both",
+    "addr": "127.0.0.1",
+    "multiTenant": false,
+    "requireSsl": false,
+    "idleTimeoutSeconds": 16,
+    "saslMechanisms": "ANONYMOUS",
+    "maxFrameSize": 16384,
+    "requireEncryption": false,
+    "host": "0.0.0.0",
+    "cost": 1,
+    "role": "normal",
+    "http": false,
+    "maxSessions": 32768,
+    "authenticatePeer": false,
+    "type": "org.apache.qpid.dispatch.listener",
+    "port": "amqp",
+    "identity": "listener/0.0.0.0:amqp",
     "name": "listener/0.0.0.0:amqp"
   }
 ]
 ----
 
 For more information about `qdmanage`, see the {qdmanageManPageLink}.
 
+== Closing a connection
+
+If a consumer is processing messages too slowly, or has stopped processing messages without settling their deliveries, you can close the connection. By closing the connection, the "stuck" deliveries will be released, and they can be redelivered to a different consumer.
```

Comment:
Eliminate dangling modifier and adopt present tense:

"By closing the connection, the "stuck" deliveries will be released" --> "When you close the connection, the "stuck" deliveries are released"